Merge Develop to Master

tkanchan opened the pull request:
Before issuing a pull request, please see the contributing page.

Commit:
Updated Spikes code changing file name form tensorflowtestingfun to TensorFlowForAutonomousBlueAndRed and also implemented it into road runner and implement it into every roadunner coad

Commit:
Rebase cs008-spikes-chatgpt onto develop branch.

Commit:
Test road runner and tensorflow code at 75% confident and change screen width to 2200.

Win10MC committed:
new drone launcher code

3DCoded committed:
fix android version

3DCoded committed:
Merge remote-tracking branch 'origin/develop' into CS019-apriltags

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Karen.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumOpMode.java

Commit:
added the new training model and changed the code acordingly

Commit:
added the new training model and changed the code acordingly

Win10MC committed:
inverse kinematics!

Win10MC committed:
Merge remote-tracking branch 'origin/CS023-NewArm' into CS023-NewArm

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Karen.java

Win10MC committed:
isolate arm code from everything else

3DCoded committed:
fixed git

Win10MC committed:
fix some warnings and remove autonomous from newArm branch

Win10MC committed:
Merge remote-tracking branch 'origin/CS023-NewArm' into CS023-NewArm

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumOpMode.java

Win10MC committed:
Revert "fixed git"

This reverts commit a946c604

3DCoded committed:
Merge remote-tracking branch 'origin/CS019-apriltags' into CS100-autonomous-develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java

3DCoded committed:
implement placeholder pixel placing

3DCoded committed:
add SpikePixel OpModes

Commit:
Remove extra spacing from Claw.java.

Commit:
Remove extra spacing from Arm.java.

3DCoded committed:
tweak some values after further simulation with MeepMeep

Enriquepe committed:
Change logic and math

Enriquepe committed:
Merge branch 'CS019-apriltags' of https://github.com/BruinBots/Centerstage into CS019-apriltags

Commit:
Merging CS023-NewArm branch onto develop branch

Commit:
Merge CS023-NewArm branch to develop branch.

Win10MC committed:
comment out broken code from merge

3DCoded committed:
implement spike movement

3DCoded committed:
Merge branch 'develop' into CS100-autonomous-develop

# Conflicts:
#	MeepMeepTesting/src/main/java/com/example/meepmeeptesting/MeepMeepTesting.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Karen.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumOpMode.java

3DCoded committed:
upgrade android studio

3DCoded committed:
strip OpMode from AprilTags

3DCoded committed:
Merge remote-tracking branch 'origin/CS019-apriltags' into CS100-autonomous-develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AprilTags.java

3DCoded committed:
AprilTags broken

3DCoded committed:
fix apriltags i think

3DCoded committed:
implement experimental algorithms

3DCoded committed:
implement experimental algorithms

Commit:
added the new training model and changed the code acordingly
1. added new function to elemante the very big boxes
2. new confidence is 0

Commit:
added the new training model and changed the code acordingly
1. added new function to elemante the very big boxes
2. new confidence is 0

Commit:
Merge branch 'CS008-spikes-chatgpt' of github.com:BruinBots/Centerstage into CS008-spikes-chatgpt

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/TensorFlowForAutonomousBlueRed.java

3DCoded committed:
experimental arm and scoop movements

Win10MC committed:
add new methods for claw fingers

Win10MC committed:
claw controls

3DCoded committed:
flip y value with April Tags to fix issue

Commit:
Remove extra spacing from build.dependencies.gradle.

Commit:
Merge remote-tracking branch 'origin/CS008-spikes-chatgpt' into CS008-spikes-chatgpt

3DCoded committed:
implement finishPixel

Win10MC committed:
testing op mode for claw testing and a bunch of other claw finger things

Win10MC committed:
testing op mode for claw testing and a bunch of other claw finger things

3DCoded committed:
stuff

3DCoded committed:
Merge branch 'CS006-drone' into develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Karen.java

3DCoded committed:
merge

Commit:
added the new training model and changed the code acordingly
1. added new function to filter out the largest shape, area, confidence

Commit:
added the new training model and changed the code acordingly
1. added new function to filter out the largest shape, area, confidence

Win10MC committed:
testing op mode for claw testing and a bunch of other claw finger things

3DCoded committed:
implement AprilTagsUpdated

3DCoded committed:
updated april tags

Commit:
Update to use redTFod function.
Remove movement from RedNearSpike.

Win10MC committed:
claw code that has set positions

Commit:
Modify TensorFlowForAutonomousBlueRed to support both blue and red tflite models.

Commit:
Modify TensorFlowForAutonomousBlueRed to support both blue and red tflite models.

Commit:
Switch between redTFod and blueTfod.

Commit:
Update blueTfod and redTfod logics.

Commit:
Update blueTfod and redTfod logics.

Commit:
Update all automation code for blue and red with TEAM_PROP_COLOR.

Commit:
Update all automation code for blue and red with TEAM_PROP_COLOR.

Win10MC committed:
drone/hanger/dropper code added

3DCoded committed:
fix april tags!!

Commit:
Remove top 3 and 10 selections.

Commit:
Check updateRecognitions size.

Win10MC committed:
testing op mode

Win10MC committed:
Merge branch 'CS024-NewClaw' of https://github.com/BruinBots/Centerstage into CS024-NewClaw

3DCoded committed:
fix negative calculation with april tags

Win10MC committed:
arm/claw/drone/dropper/hanger code

3DCoded committed:
calibrate scoop

3DCoded committed:
calibrate scoop

3DCoded committed:
april tags work!!

3DCoded committed:
experimental pixel placment (minus claw)

Win10MC committed:
teleop code

3DCoded committed:
claw

Win10MC committed:
more teleop changes

Commit:
Fix the indexOutOfBound error by returning null from updatedrecognition.

Commit:
Introduce getSide paramter to select blue or red for telemetryTfod.

Commit:
Introduce getSide paramter to select blue or red for telemetryTfod.

Commit:
Combine three criteria as Team prop
 1. select and sort by DESC for top three confidences
 2. Select the label with largest area.
 3. Select the label that closer to square by getWidth/getHeight and sorted by the result ASEC and selec the top

 to telemetryTfod.

Commit:
Comment out bot.claw.openClaw() and bot.claw.closeBothClaw() to prevent build fail.

Commit:
Merge branch 'CS008-spikes-chatgpt' of https://github.com/BruinBots/Centerstage into CS008-spikes-chatgpt

tkanchan committed:
Update AndroidManifest.xml

Update version to 9.0.1 to solve control hub config issue.

Win10MC committed:
new motor and update manifest

Commit:
updated manifestests

tkanchan reviewed (commented):
approved

Win10MC committed:
new motor changes

3DCoded committed:
fix android manifest version

3DCoded commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java:
We shouldn't merge this

3DCoded commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoTest.java:
This shouldn't be merged

3DCoded commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumOpMode.java:
I don't think we should merge this

3DCoded reviewed (approved):
This looks good.

3DCoded committed:
fix code

3DCoded committed:
Merge remote-tracking branch 'origin/develop' into CS100-autonomous-develop

# Conflicts:
#	FtcRobotController/src/main/AndroidManifest.xml
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Karen.java

3DCoded committed:
new arm

tkanchan committed:
Merge pull request #3 from BruinBots/CS008-spikes-chatgpt

Cs008 spikes chatgpt

3DCoded committed:
add camera stream to dashboard

3DCoded committed:
preliminary pixel placement code

tkanchan committed:
Merge branch 'develop' into CS024-NewClaw

tkanchan committed:
Merge pull request #2 from BruinBots/CS024-NewClaw

Merge newclaw branch into develop

Commit:
Fixed error from NewArm merge to develop.

3DCoded committed:
pixel placing works!!

3DCoded committed:
pixel placing works!!

3DCoded committed:
new auto

3DCoded committed:
Merge branch 'CS100-autonomous-develop' into CS024-NewClaw

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Arm.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoTest.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Claw.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Dropper.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/InOutTake.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Karen.java

3DCoded committed:
Make code usable

3DCoded committed:
reverse arm

tkanchan committed:
Merge branch 'develop' into CS100-autonomous-develop

tkanchan committed:
Merge pull request #4 from BruinBots/CS100-autonomous-develop

Merge Autonomous

3DCoded committed:
Merge branch 'develop' of https://github.com/BruinBots/Centerstage into develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/InOutTake.java

3DCoded committed:
remove errors

3DCoded committed:
Merge branch 'CS024-NewClaw' into develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Arm.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/ArmTestMode.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Claw.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Dropper.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/InOutTake.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Karen.java

3DCoded committed:
fix bugs

Commit:
reverse clawUpperFinger

3DCoded committed:
Merge pull request #5 from BruinBots/CS024-NewClaw

TeleOp Code

3DCoded committed:
update autonomous

3DCoded committed:
change coordinates

3DCoded committed:
add arm safety

3DCoded committed:
fix mecanumopmode

3DCoded committed:
Merge branch 'develop' into CS100-autonomous-develop

3DCoded committed:
now it compiles

3DCoded committed:
new arm values

3DCoded committed:
Merge branch 'CS024-NewClaw' into develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Arm.java

3DCoded committed:
merge newClaw

Enriquepe committed:
Change logic and math

3DCoded committed:
fix Karen issues

3DCoded committed:
fix lower finger

Commit:
test tensorflow

3DCoded committed:
rr not working

Win10MC committed:
arm changes

Win10MC committed:
arm changes

Win10MC committed:
arm changes

tkanchan committed:
Merge pull request #6 from BruinBots/CS024-NewClaw

Cs024 new claw

Enriquepe committed:
Change logic and math

3DCoded committed:
fix stuff

3DCoded committed:
commit message

tkanchan committed:
Update MecanumOpMode.java

Commit:
Merge branch 'CS100-autonomous-develop' of https://github.com/BruinBots/Centerstage into CS100-autonomous-develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoTest.java

Brokenpoet committed:
Added drive speed ramping.

Win10MC committed:
sort out some stuff that happened when someone didnt pull before pushing

Win10MC committed:
Merge remote-tracking branch 'origin/CS024-NewClaw' into CS024-NewClaw

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Arm.java

Win10MC committed:
offset for inverse kinematics

Brokenpoet committed:
Added an emergency arm lowering mode in the Arm Class.
It will activate when Anna's controller (gamepad1) has both bumpers simultaneously pressed.
It does not care about the scoop, or the claw, or anything.  It just lowers the arm and resets the encoder.  It is Dangerous.

Brokenpoet committed:
Merge remote-tracking branch 'origin/CS024-NewClaw' into CS024-NewClaw

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Arm.java

Win10MC committed:
additional safety checks

Win10MC committed:
Merge branch 'CS024-NewClaw' of https://github.com/BruinBots/Centerstage into CS024-NewClaw

Win10MC committed:
revert back to old drive mode and make emergency lower work

3DCoded committed:
Merge branch 'develop' into CS100-autonomous-develop

3DCoded committed:
clean up RR

3DCoded committed:
Implement pure RR solution to backdrop alignment

3DCoded committed:
add backdrop offset

3DCoded committed:
make code compile

3DCoded committed:
Merge branch 'CS100-autonomous-develop' into HARRISONBURG_COMP

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AprilTagsAutonomous.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BlueFarAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BlueNearAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/RedFarAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/RedNearAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoOpModes/BlueFar/BlueFarFull.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoOpModes/BlueNear/BlueNearFull.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoOpModes/RedFar/RedFarFull.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoOpModes/RedNear/RedNearFull.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoTest.java

Commit:
Update backdrop code to slower down the robot.

3DCoded committed:
Copied Park Modes

3DCoded committed:
remove tensorflow

3DCoded committed:
fix

Commit:
Merge branch 'CS100-autonomous-develop' of https://github.com/BruinBots/Centerstage into CS100-autonomous-develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java

Brokenpoet committed:
Merge remote-tracking branch 'origin/HARRISONBURG_COMP' into HARRISONBURG_COMP

Brokenpoet committed:
Merge remote-tracking branch 'origin/CS100-autonomous-develop' into HARRISONBURG_COMP

Brokenpoet committed:
Added Dashboard to Teleop gradle dependencies

3DCoded committed:
pure RR again

Commit:
Merge branch 'CS100-autonomous-develop' of https://github.com/BruinBots/Centerstage into CS100-autonomous-develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoTest.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/TensorFlowForAutonomousBlueRed.java

3DCoded committed:
add tensorflow again

3DCoded committed:
Merge remote-tracking branch 'origin/CS100-autonomous-develop' into CS100-autonomous-develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/TensorFlowForAutonomousBlueRed.java

Commit:
Merge branch 'HARRISONBURG_COMP_AUTO' into CS100-autonomous-develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AprilTagsAutonomous.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoTest.java

Commit:
Merge remote-tracking branch 'origin/CS100-autonomous-develop' into CS100-autonomous-develop

Brokenpoet committed:
Merge remote-tracking branch 'origin/CS100-autonomous-develop' into HBURG-VINCE

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AprilTagsAutonomous.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BaseAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BlueFarAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BlueNearAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/RedFarAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/RedNearAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoOpModes/BlueFar/BlueFarFull.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoOpModes/BlueNear/BlueNearFull.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoOpModes/RedFar/RedFarFull.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoOpModes/RedNear/RedNearFull.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoTest.java

Commit:
merge from CS100-autonomous-develop2

Commit:
Merge branch 'CS100-autonomous-develop' into HBURG-VINCE

Commit:
Fix BaseAuto merge error.

3DCoded committed:
change parking logic

Commit:
Add missing files from AutoOpModes folder.

Commit:
Fixed error on conflict from AprilTagsAutonomous.

Brokenpoet committed:
Updated starting positions to y:62 (y:-62) in Far/NearAuto
Added StartPark in BaseAuto to move 2" away from the wall at start of all auto modes
Added startPark to all ParkOnly Modes

Brokenpoet committed:
Merge remote-tracking branch 'origin/HBURG-VINCE' into HBURG-VINCE

Commit:
Adding orb1-14-24.tflite and RedSphere1.tflite for Tensorflow models.

Commit:
Merge branch 'HBURG-VINCE' of https://github.com/BruinBots/Centerstage into HBURG-VINCE

3DCoded committed:
change parking y to +- 61 from +- 62

3DCoded committed:
Figure out parking and pixel placement/backdrop

3DCoded committed:
backdrop

Commit:
Adjust BlueNearSpikePixel.

Commit:
Adjust BlueNearSpikePixel and work.

Commit:
Adjust BlueNearSpikePixel and work.

Commit:
Adjust BlueNearSpikePixel and work.

tkanchan committed:
Merge pull request #7 from BruinBots/HBURG-VINCE

Hburg vince

Commit:
Need Adjust at the field for BlueFar, RedFar, RedNear.

3DCoded committed:
Clean up code, increase max velocity in DriveConstants to finish autonomous under 30 seconds

3DCoded committed:
Set blue=false for the red opmodes

3DCoded committed:
Set blue=false for the red opmodes...again

3DCoded committed:
attempt to fix coordinates to not hit the poles

3DCoded committed:
bluenear works

3DCoded committed:
fix bugs

3DCoded committed:
clean up code

3DCoded committed:
Merge branch 'HBURG-VINCE' into develop

3DCoded committed:
Merge branch 'develop' into CS100-autonomous-develop

# Conflicts:
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BlueFarAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/BlueNearAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/RedFarAuto.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Autonomous/AutoBases/RedNearAuto.java

3DCoded committed:
implement coordinates.properties

3DCoded committed:
more testing

3DCoded committed:
implement logic for center/left/right spike mark to improve efficiency

3DCoded committed:
reimplement inverse kinematics into autonomous

3DCoded committed:
test

3DCoded committed:
test

3DCoded committed:
fix tensorflow bug; "none" doesn't exist anymore, now use "center", also give tensorflow more time to think (40ms -> 100ms)

3DCoded committed:
fix tensorflow bug; "none" doesn't exist anymore, now use "center", also give tensorflow more time to think (40ms -> 100ms)

3DCoded committed:
clean up code a bit

3DCoded committed:
clean up code more

3DCoded committed:
clean up code more, re-implement coordinates.properties to BlueNearAuto

3DCoded committed:
more testing, spikes work! inverse kinematics in autonomous works!

3DCoded committed:
CRITICAL: add sleep before inverse kinematics kicks in

3DCoded committed:
add more timing safety

3DCoded committed:
change backdrop logic

3DCoded committed:
code review

Win10MC committed:
add distance sensor code to develop

Win10MC committed:
cube the input and add stuff for distance sensor

3DCoded committed:
Merge branch 'develop' into CS100-autonomous-states

3DCoded committed:
implement distance sensor into Backdrop

3DCoded committed:
pixel placement on backdrop + distance sensor works and is tested

3DCoded committed:
fine tune autonomous

3DCoded committed:
fix autonomous

Commit:
Fixing issue with autonomouse RedNearFull not fully function because too much slee().
Updated TesorFlow code.

3DCoded committed:
undo coordinate change to fix it again

3DCoded committed:
change coordinates again

3DCoded committed:
undo backdrop change to make it work again

3DCoded committed:
change ownership of bot in autonomous to BaseAuto from Backdrop

3DCoded committed:
fix RR error in BaseAuto

3DCoded committed:
attempt to fix distance sensor

3DCoded committed:
all full opmodes work now

Enriquepe committed:
Change logic and math

3DCoded committed:
add sanity checks, implement safeSleep

3DCoded committed:
fix coordinates

Commit:
Update TensorFlowForAutonomousBlueRed logic for screenwidth when camera is on the left side of the robot.

3DCoded committed:
remove compensation from coordinates.properties, as error was traced to deadwheel issues on hardware; will need further testing

3DCoded committed:
revise sanity check, now if distance is > 22, then it will refuse to move more than 22 inches to save the robot from potentially breaking the arm

3DCoded committed:
Merge remote-tracking branch 'origin/CS100-autonomous-states' into CS100-autonomous-states

3DCoded committed:
change parking logic to turn in order to drive rather than strafe, saving time and increasing precision

3DCoded committed:
increase angular velocity and angular acceleration for efficiency

3DCoded committed:
update coordinates for spikeenter and spikerelative's

Enriquepe committed:
Change logic and math

3DCoded committed:
change coordinates

3DCoded committed:
Merge branch 'develop' into driving_tune

3DCoded committed:
change coordinates

3DCoded committed:
add offset in order to drop pixel on the spike mark

Enriquepe committed:
Change logic and math

Commit:
Fixed TensorFLow null pointer exception

3DCoded committed:
Merge branch 'CS100-autonomous-states' into develop

3DCoded committed:
fix illegal initialization behavior

3DCoded committed:
remove tensorflow

3DCoded committed:
Merge branch 'develop' into CS100-autonomous-states

3DCoded committed:
fix bug

3DCoded committed:
make scoop up before turning for park

3DCoded committed:
clean up tensorflow code

3DCoded committed:
add some more distance sensor logic; now should take ~1s to compute
